### PR TITLE
remove Remote from mongodb links. update master link to beta.13

### PR DESCRIPTION
**Pull request by MongoCaleb**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-12433

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/12433/index.html

### See Also
https://jira.mongodb.org/browse/RJS-689
https://github.com/realm/realm-js/pull/3175/commits/0cb2199e7481cb11789bc5de7426721511b9d663

### CODE REVIEW

File: source/react-native/mongodb.txt [hunk 1]
```diff
@@ -73,7 +73,7 @@ Insert Multiple Documents
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 You can insert multiple documents at the same time using
-:js-sdk:`collection.insertOne() <Realm.RemoteMongoDBCollection.html#insertMany>`.
+:js-sdk:`collection.insertOne() <Realm.MongoDBCollection.html#insertMany>`.
```
Comment (ccho-mongodb) [inline source/react-native/mongodb.txt:76]: You fixed this in the other instance, but not this one 😛

**Review by ccho-mongodb — APPROVED** [review]
One change required, but otherwise LGTM.

File: source/react-native/mongodb.txt (hunk 1 above)
Comment (MongoCaleb) [reply source/react-native/mongodb.txt:76]: good catch!

File: conf.py [hunk 2]
```diff
@@ -108,7 +108,7 @@
     'kotlin-sdk': ('https://docs.mongodb.com/realm-sdks/kotlin/10.0.0-BETA.6/%s', ''),
     'swift-sdk': ('https://docs.mongodb.com/realm-sdks/swift/10.0.0-beta.5/%s', ''),
     'objc-sdk': ('https://docs.mongodb.com/realm-sdks/objc/10.0.0-beta.5/%s', ''),
-    'js-sdk': ('https://docs.mongodb.com/realm-sdks/js/10.0.0-beta.9/%s', ''),
+    'js-sdk': ('https://docs.mongodb.com/realm-sdks/js/10.0.0-beta.13/%s', ''),
```
Comment (mongomoe) [inline conf.py:111]:
```suggestion
    'js-sdk': ('https://docs.mongodb.com/realm-sdks/js/%s', ''),
```

will default to the latest version

**Review by mongomoe — APPROVED** [review]
LGTM 👍

**Pull request merged**